forc-plugins/forc-client/tests/deploy.rs

```diff
@@ -341,7 +341,7 @@ async fn test_simple_deploy() {
     node.kill().unwrap();
     let expected = vec![DeployedContract {
         id: ContractId::from_str(
-            "50fe882cbef5f3da6da82509a66b7e5e0a64a40d70164861c01c908a332198ae",
```

Comment:
I've turned this PR back into draft, and am pulling out all the changes so that we can see which crates can be updated with no code changes, and which ones end up doing changes like these.

Unfortunately it's a bit time consuming because you end up having to wait for the whole `ci.yml` to finish before the next 'edit -> test -> debug` cycle